HIRING.md

```diff
@@ -0,0 +1,53 @@
+
+## Structure
+
+### Use about 5 minutes
+- Introduce yourself
+- Introduce B&B (Office locations, what we do)
+- Introduce the position they are interviewing for
+
+### Use about 10-15 minutes
+- Pick a role from their CV that looks interesting (or their final project if a graduating student)
+- Ask them to talk about the role; their responsibilities; things that went well; things that went badly; why they went badly. Here you are trying to assess how they are as a worker -- are they a leader, or a follower? Are they outspoken or quiet? Do they work well in a team, or alone? You are also finding out how good they are at communicating -- can you understand what they actually did; can they answer clarifying questions about what they did? This has the benefit of warming up the candidate; talking about something they have done is easy.
+
+### Use about 20-30 minutes
+- Ask a few technical questions. Examples are found below.
+- Here you are trying to assess what they know. At the same time you can assess if they are honest in what they don’t know. Push them to the limits of their knowledge but don’t waste time if they don’t know something; change your questioning to their knowledge so that they are suitably challenged.
+- By the end you should have a good sense of their technical skills but also a good impression of how they think, how they solve problems and how they operate under pressure.
+
+### Use about 10-15 minutes
+- Talk about the candidate. How do they keep up to date with developments in their chosen field? Do they write blogs, do they tweet?
+- What does the candidate think would be their perfect role at B&B?
+- If they are going to have to change country, what do they know about that country? Why do they want to move.
+- The purpose here is to understand more about the candidate; their aspirations and what drives them.
+
+### Use about 5 minutes
+- Answer any questions they have. Be honest. Don’t talk for too long.
+- Let the answers flow like a conversation.
+- Thank the candidate for their time and let them know what the next step will be.
+
+## General Questions
+- Do you know who's Mattt Thompson?
+- Do you know who's Loren Britcher?
+- Have you heard of ASIHTTPRequest?
+- Have you heard of Three20?
+- For how long you've been an iOS user?
+- What's your favorite native iOS app and why?
+- What's your favorite third-party iOS app and why?
+- What do you think makes a great app?
+
+## Technical Questions
+- What's the superclass of UITableView?
```

Comment:
Since my comments disappeared: I don't like this section. Feels like superficial trivia, like we're trying to trick the candidate into revealing they're not actually developers, but a cab driver or a builder in disguise. 😂